inst/css/custom.css

```diff
@@ -85,3 +85,12 @@
   height:600px;
   width:100%
 }
+
+/* bootstrap rules */
+.bootstrap-switch .bootstrap-switch-primary {
+    background: #337ab7 !important;
+}
+
+.bootstrap-switch.bootstrap-switch-focused {
+    border-color: #2e6da4 !important;
+}
```

Comment:
Are we overwriting theme colors? With hard coded ones?

Reply:
I did not like it either but to maintain consistency with the other buttons, I modified the CSS property of `'bootstrap-switch_primary'.` While there is a possibility to alter the color using the `'onStatus' `option in `'shinywidget::switchInput',` the available choices are confined, as detailed in the [list provided here](https://getbootstrap.com/docs/4.0/utilities/colors/). Given that 'switchInput' is also employed in 'tm_g_bivariate', I opted for this approach.

As switchInput is also used tm_g_bivariate as welll so I went with this option.

Reply:
I was under the impression that we should be using consistent colors throughout. Are status colors not enough to achieve this?

Reply:
I share @chlebowa opinion. Can we instead utilise primary, secondary, etc. everywhere?
I think what you do is against [this](https://github.com/insightsengineering/teal/issues/735) issue?

Reply:
I'd suggest to explore these (general) issues, making patches here and there:

- https://github.com/insightsengineering/teal/issues/735
- https://github.com/insightsengineering/teal/issues/802
- https://github.com/insightsengineering/teal/issues/762


I think we should first solve general styling of teal apps and then fix wherever style would be incompatible with standard one.

Reply:
Certainly, I concur. I will address color consistency as a component of these distinct tasks and undo the color modifications from the following pull requests:

Issue #735 - GitHub link: https://github.com/insightsengineering/teal/issues/735
Issue #802 - GitHub link: https://github.com/insightsengineering/teal/issues/802
Issue #762 - GitHub link: https://github.com/insightsengineering/teal/issues/762